shimmingtoolbox/cli/image.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*
+
+import click
+import os
+import nibabel as nib
+
+from shimmingtoolbox.image import concat_data
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+AXES = ['0', '1', '2', '3', '4']
+
+
+@click.group(context_settings=CONTEXT_SETTINGS,
```

Comment:
I know it's a "click" thing, but the usage shows up like this:
~~~
Usage: st_image concat [OPTIONS] INPUT...
~~~

whereas in fact, INPUT could go _before_ the options (which i find more intuitive). I would prefer having the INPUT listed before, but it's not a big deal...

Reply:
I had noticed the same thing, however, the layout does not seem to be customizable yet according to the [doc](https://click.palletsprojects.com/en/7.x/documentation/)

> Click makes it very easy to document your command line tools. First of all, it automatically generates help pages for you. While these are currently not customizable in terms of their layout, all of the text can be changed.